### ISSUE TITLE
Uncaught TypeError: obj.find(...).andSelf is not a function : /taxonomies/:id/edit

### ISSUE THREAD

**Issue by aguayma** [spree/spree#7503]
Currently running into issue where getting the follow response on console.

``` js
jquery.jstree.self-4133052….js?body=1:703 Uncaught TypeError: obj.find(...).andSelf is not a function
```

Receive this error when page initially loads, even with a defined Taxonomy. 

gemfile: https://gist.github.com/joshRpowell/3fab3f4a248b33754fe094af96dba1f0
backend/all.js: https://gist.github.com/joshRpowell/6c00bf2ae6c17ddd3a48d4b7543989e7

Included a video for reference. 
https://monosnap.com/file/oUezblYaFQDgkVJll7KitSQKTDSvSG

@damianlegawiec I noticed you were working on the taxonomy.js.coffee file, and think this may be related in some way.

**Comment by damianlegawiec** [comment]
Hi @aguayma - my recent work on taxonomy tree is only in `master` branch, not in `3.1` release but we'll look into it - thank you for supplying so many resources with your issue 👍

**Comment by josyan** [comment]
Replacing `andSelf` by `addBack` resolve the problem for me, but I think it depends on the version of `jQuery` you are using. `andSelf` was deprecated in `jQuery` 1.8, and removed in `jQuery` 3.0.

**Comment by bbonislawski** [comment]
It was fixed on master with #8308 .
I'll backport it to older versions. Thanks for reporting issue!
